docs/cases/cases-overview.asciidoc

```diff
@@ -68,3 +69,32 @@ To view a case, click on its name. You can then:
 * Refresh the case to retrieve the latest updates
 
 NOTE: Comments can also contain Markdown syntax and Timeline links.
+
+[float]
+[[cases-lens-visualization]]
+== Add a Lens visualization
+
+beta[]
+
+Add a Lens visualization to your case to portray event and alert data through charts and graphs.
+
+IMPORTANT: Set an absolute time range for your visualization. This ensures your visualization doesn't change over time after you save it to your case and provides important context for other viewers.
+
+[role="screenshot"]
+image::images/add-vis-to-case.gif[Shows how to add a visualization to a case]
+
+To add a Lens visualization to your case within a comment:
+
+. Click the *Visualization* button. The Add Visualization modal appears.  
```

Comment:
To match the sentence-case label as seen in the GIF. Also, "modal" seems like dev jargon to me, vs. "dialog" or "dialog box," either of which appear to be more common in other Elastic docs.
```suggestion
. Click the *Visualization* button. The Add visualization dialog appears.  
```